bin/plan_check

```diff
@@ -188,6 +199,7 @@ def execute_init_and_plan(
         skip_iam: bool,
         print_diff: bool,
         with_colors: bool,
+        output_directory: Optional[str],
```

Comment:
Is this actually Optional? There is no default value for it. Also I think optional args should be at the end of the list

Reply:
Should it be a necessary option? I have no idea what default value it can have. `/tmp/tfplan/`?

Reply:
Yea I think it should be optional. Just meant it should be `output_directory: Optional[str] = null`